test/recipes/90-test_memleak.t

```diff
@@ -10,6 +10,10 @@
 use OpenSSL::Test;
 
 setup("test_memleak");
+
+plan skip_all => "MacOS currently doesn't support leak sanitizer"
```

Comment:
Is isn't? https://clang.llvm.org/docs/LeakSanitizer.html

Reply:
I guess it depends on which version of clang/llvm you have :(

Reply:
Yup.